addon/components/offering-form.js

```diff
@@ -441,16 +441,17 @@ export default class OfferingForm extends Component {
 
     for (let i = 0; i < offerings.length; i++) {
       const {startDate, endDate, learnerGroups, learners} = offerings[i];
-      let {room} = offerings[i];
+      let { room } = offerings[i];
       for (let j = 0; j < learnerGroups.length; j++) {
         const learnerGroup = learnerGroups[j];
         const defaultLocation = learnerGroup.get('location');
         if (isPresent(defaultLocation)) {
           room = defaultLocation;
         }
+        const url = learnerGroup.get('url') ;
```

Comment:
Can just be `learnerGroup.url` or `const { url } = learnerGroup`.

Reply:
handled.